deps/scratch.jl

```diff
@@ -83,39 +86,35 @@ elseif Sys.ARCH === :x86_64
     fc *= " -m64"
 end
 
-flags = [
-    # OpenSpecFun build flags
-    "ARCH=\"$(Sys.ARCH)\"",
-    "CC=\"$cc\"",
-    "FC=\"$fc\"",
-    "USECLANG=$(Int(use_clang))",
-    "USEGCC=$(Int(!use_clang))",
-    "USE_OPENLIBM=$(Int(use_openlibm))",
-    "CFLAGS=\"-O3 -std=c99\"",
-    "FFLAGS=\"-O2 -fPIC\"",
-    "LDFLAGS=\"-L$(libdir(openspecfun)) -Wl,-rpath,'\$\$ORIGIN' -Wl,-z,origin\"",
-    # Make flags
-    "DESTDIR=\"\"",
-    "prefix=$(depsdir(openspecfun))",
-    "libdir=$(libdir(openspecfun))",
-    "shlibdir=$(libdir(openspecfun))",
-    "includedir=$(includedir(openspecfun))",
-    "O="
-]
-
 provides(Sources, URI("https://github.com/JuliaLang/openspecfun/archive/v$OSF_VERS.tar.gz"),
-         openspecfun)
+         openspecfun, unpacked_dir="openspecfun-$OSF_VERS")
 
 provides(BuildProcess,
     (@build_steps begin
         GetSources(openspecfun)
         CreateDirectory(builddir(openspecfun))
         @build_steps begin
-            ChangeDirectory(builddir(openspecfun))
+            ChangeDirectory(joinpath(srcdir(openspecfun), "openspecfun-$OSF_VERS"))
             FileRule(joinpath(libdir(openspecfun), "libopenspecfun." * Libdl.dlext),
                 @build_steps begin
                     CreateDirectory(libdir(openspecfun))
-                    `$MAKE_CMD install $flags`
+                    ```
+                    $MAKE_CMD install
+                        ARCH="$(Sys.ARCH)"
+                        CC="$cc"
+                        FC="$fc"
+                        USECLANG=$(Int(use_clang))
+                        USEGCC=$(Int(!use_clang))
+                        USE_OPENLIBM=$(Int(use_openlibm))
+                        CFLAGS="-O3 -std=c99"
+                        FFLAGS="-O2 -fPIC"
+                        LDFLAGS="-L$(libdir(openspecfun)) -Wl,-rpath,'\$\$ORIGIN' -Wl,-z,origin"
+                        DESTDIR=""
+                        prefix=""
```

Comment:
why is this dropping the prefix?

Reply:
It's not needed when `libdir` and `includedir` are fully qualified paths.